lib/puppet/face/config.rb

```diff
@@ -71,7 +80,18 @@
                      _("New environment loaders generated from the requested section.")) do
         # And now we can lookup values that include those from environments configured from
         # the requested section
+
+        if @default_section
+          Puppet.warning (_(<<-EOM) % { section_name: options[:section] } ).chomp
+No section specified, using '%{section_name}'.
```

Comment:
I'm a bit concerned that "using <section_name>" will be a bit ambiguous to translate here, especially since I initially read it as "you didn't specify a section via <section_name>" instead of "you didn't specify a section, so we are going to use <section_name>". Perhaps "No section specified; defaulting to '%{section_name}'", instead?

Reply:
I agree with the ambiguity, but thought of another phrase as I went to edit this.  Do you like "defaulting to X" or "using default section X" better?

Reply:
I have no strong opinion on which of those two to use. They're both better than what they'd be replacing. :)